storage/boltdb/user.go

```diff
@@ -16,8 +21,11 @@ type User struct {
 
 // CreateUser calls bolt database instance to create user
 func (bdb *Client) CreateUser(user User) error {
-	return bdb.DB.Update(func(tx *bolt.Tx) error {
-		b := tx.Bucket([]byte("users"))
+	return bdb.db.Update(func(tx *bolt.Tx) error {
+		b, err := tx.CreateBucketIfNotExists([]byte("users"))
```

Comment:
same as files but if this is ok to be hardcoded might want to move it to a const